src/resources/nfts/index.ts

```diff
@@ -20,8 +24,20 @@ const POLYGON_ALLOWLIST_STALE_TIME = 600000; // 10 minutes
 export const nftsQueryKey = ({ address }: { address: string }) =>
   createQueryKey('nfts', { address }, { persisterVersion: 1 });
 
+export const nftListingQueryKey = ({
+  contractAddress,
+  tokenId,
+  network,
+}: {
+  contractAddress: string;
+  tokenId: string;
+  network: Omit<Network, Network.goerli>;
+}) => createQueryKey('nftListing', { contractAddress, tokenId, network });
```

Comment:
No cache on this one? Just double checking, makes sense to me since it's like live data.

Reply:
yeah thats what i was thinking!